### DISCUSSION TITLE
[Feature] Add option to choose file types to upload (mobile)

### DISCUSSION THREAD

**Discussion by FibreTTP** [immich-app/immich#5987] [Feature Request]
### The feature

The Google Pixel camera app (at the very least) has recently begun saving *both* the RAW and JPEG versions of a photo in the normal `DCIM/Camera` directory. See the filename syntax here: https://github.com/immich-app/immich/discussions/2479#discussioncomment-7945896

Thus, those who continue to save RAW+JPG will have now the RAW versions of the photos they take uploaded, even if they had the previous `Pictures/RAW` Album unselected (for various reasons, such as RAWs **not automatically stacking with JPEGs** <- very annoying).

An option should be added in the Album backup selector to white/blacklist file extensions so one can save RAWs but prevent them from being uploaded.

And yes, this is 100% Google's fault for making breaking changes with basically no benefit, and seemingly without telling anyone.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by FibreTTP** [comment]
By commenting again, I hope to bring attention to this request.

**Reply by bo0tzz** [reply]
We see all the requests, no need to bump them.
